src/components/menus/TertiaryNav/TertiaryNav.sass

```diff
@@ -68,5 +68,5 @@
 	.TertiaryContent
 		flex: 1
 		overflow: inherit
-
+		max-height: calc(100vh - 255px)
```

Comment:
I like where this is going! Will this account for all cases? If not, we could add the ability to pass down `classnames` which would allow us to set or override this on a case by case basis.